src/code/sys_cfb.c

```diff
@@ -6,7 +6,7 @@ uintptr_t sSysCfbEnd;
 void SysCfb_Init(s32 n64dd) {
     u32 screenSize;
     uintptr_t tmpFbEnd;
-
+#ifdef EXPANSION
     if (osMemSize >= 0x800000) {
```

Comment:
I don't think this is needed, I'd change ``osMemSize`` instead it looks more correct this way

Reply:
osMemSize doesnt behave properly on gcc so my work around solution is to disable 8mb mode entirely

Reply:
oh ok, then I guess just rename the define here by ``ENABLE_EXPANDED_MEMORY`` and it'll be good 👍

Reply:
decomp found a fix in lib ultra, testing it now

Reply:
nice! hope it works